### DISCUSSION TITLE
大佬们教教我：怎么让mosdns返回我指定的CNAME记录？

### DISCUSSION THREAD

**Discussion by aa51513** [IrineSistiana/mosdns#255] [Q&A] [answered]
比如我查询www.example.com，我想让mosdns拦截后，把结果变成web.example.com对应的IP地址

我应该怎么配置？

**Comment by IrineSistiana** [comment, marked as answer]
实现不了。即使改了cname mosdns只是个转发器也不会去顺着cname找IP。
